data/distro.yml

```diff
@@ -477,3 +477,21 @@
         <p>To complete setup, restart your system. Now all you have to do is <a href="https://flathub.org/">install some apps</a>!</p>
       </li>
     </ol>
+    
+ - name: "GNU Guix"
+  logo: "guix.svg"
+  info: >
+    <ol class="distrotut">
+      <li>
+        <h2>Install Flatpak</h2>
+        <p>Flatpak can be installed using GNU Guix's default repositories:</p>
+        <pre><code>
+          <span class="unselectable">$</span> # guix install flatpak
```

Comment:
Does it also install the GNOME Software (or KDE Discover) Flatpak plugin?

Reply:
I'm waiting for this question to be answered before merging.

Reply:
I do not believe so.
You can see all packages here: https://www.gnu.org/software/guix/packages/

I do not see `gnome-software-plugin-flatpak` under [/software/guix/packages/G/](https://www.gnu.org/software/guix/packages/G/)

Reply:
Hmm, I don't see the `gnome-software` package at all under /software/guix/packages/G/.

Reply:
What is the software center GUI used in GNU Guix and does it include Flatpak support?

Reply:
Hmm, it seems that [this](http://www.ocsmag.com/2016/07/05/guix-the-non-aligned-universal-package-manager/) is the package manager "GUI" that GNU Guix actually use. Does it have any Flatpak support?

Reply:
@asddsaz @lionirdeadman What is the current state of Flatpak support in GUIX?

Reply:
@asddsaz @lionirdeadman What is the current state of Flatpak support in GUIX?

Reply:
See the suggested changes.